lib/content-services/upload/directives/file-draggable.directive.ts

```diff
@@ -24,7 +24,7 @@ import { Directive, ElementRef, EventEmitter, Input, NgZone, OnDestroy, OnInit,
  * Directive selectors without adf- prefix will be deprecated on 3.0.0
  */
 @Directive({
-    selector: '[adf-file-draggable], [file-draggable]'
+    selector: '[adf-file-draggable]'
```

Comment:
Is this the correct time to start removing these selectors?

Reply:
It will not work correctly, because there's only one `named` property that enables it:

```
@Input('adf-file-draggable')
enabled: boolean = true;
```